Gopkg.lock

```diff
@@ -1,12 +1,24 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
 
 
+[[projects]]
+  branch = "master"
+  name = "github.com/Azure/go-ansiterm"
+  packages = [".","winterm"]
+  revision = "d6e3b3328b783f23731bc4d058875b0371ff8109"
+
 [[projects]]
   name = "github.com/DATA-DOG/godog"
   packages = [".","colors","gherkin"]
   revision = "4dc98b0e2b130c3c9d06868a050320e5b310d3e7"
   version = "v0.7.4"
 
+[[projects]]
+  name = "github.com/Sirupsen/logrus"
```

Comment:
-100, we cannot depend on "github.com/Sirupsen/logrus". It needs to be "github.com/sirupsen/logrus"

Reply:
I prefer to first get the pull request for #952 in and then do a docker-machine upgrade. I think the containers/image merge will occur within the next couple of days.

Reply:
@hferentschik lock file is something generated automatic, if you think it should be lowercase then this should be moved to .toml file and forced to use a specific version.

Reply:
@hferentschik Also docker using this extensively (Captial letter)

```
$ grep -nR Sirupsen .
Binary file ./.git/objects/pack/pack-84909492a1c5a924403a7939eb099cb7a29b5933.pack matches
./Gopkg.lock:17:  name = "github.com/Sirupsen/logrus"
./vendor/github.com/docker/docker/migrate/v1/migratev1.go:16:	"github.com/Sirupsen/logrus"
./vendor/github.com/docker/docker/api/common.go:14:	"github.com/Sirupsen/logrus"
./vendor/github.com/docker/docker/api/server/httputils/errors.go:7:	"github.com/Sirupsen/logrus"
./vendor/github.com/docker/docker/api/server/router/system/system_routes.go:9:	"github.com/Sirupsen/logrus"
./vendor/github.com/docker/docker/api/server/router/swarm/cluster_routes.go:9:	"github.com/Sirupsen/logrus"
./vendor/github.com/docker/docker/api/server/router/container/exec.go:10:	"github.com/Sirupsen/logrus"
./vendor/github.com/docker/docker/api/server/router/container/container_routes.go:12:	"github.com/Sirupsen/logrus"
./vendor/github.com/docker/docker/api/server/router/build/build_routes.go:15:	"github.com/Sirupsen/logrus"
./vendor/github.com/docker/docker/api/server/server.go:10:	"github.com/Sirupsen/logrus"
./vendor/github.com/docker/docker/api/server/middleware/debug.go:10:	"github.com/Sirupsen/logrus"
./vendor/github.com/docker/docker/api/server/middleware/cors.go:6:	"github.com/Sirupsen/logrus"
./vendor/github.com/docker/docker/api/server/middleware.go:4:	"github.com/Sirupsen/logrus"
./vendor/github.com/docker/docker/cli/trust/trust.go:14:	"github.com/Sirupsen/logrus"
./vendor/github.com/docker/docker/cli/flags/common.go:8:	"github.com/Sirupsen/logrus"

[...]
```

Reply:
> Also docker using this extensively (Captial letter), from there it's coming now.

 Not the later versions. See the issues with logrus here https://github.com/sirupsen/logrus. They renamed the repository causing all sorts of problems. Many dependencies have by now changed, even Docker. If you look at the pull request for #952, I am already taking care of this. We cannot have 'Sirupsen' in the dependnecy list. Not if we want to avoid a rewrite of the imports in the vendor directory. 

As said, I prefer not to do this here. Maybe I can rebase #1685 and take care of this. After all I created #1758 out of this context. I just wanted to do it in a more controlled way. @praveenkumar are you ok with closing this pull request and I add an additional commit as part of #1685

Reply:
@hferentschik Sure, I am closing this PR and you can rebase later.

Reply:
Cheers. I am updating the Dep config as we speak and will hopefully push an update shortly. You will notice that I already taken care of some parts of this, hence I reacted to stroingly. My current pull request already forces an updated version of Docker which got the logrus dependency fixed.